source/settings/core.rst

```diff
@@ -2866,12 +2866,9 @@ See :ref:`settings` for list of all setting groups.
 
 .. dovecot_core:setting:: protocols
    :default: imap pop3 lmtp
-   :values: @string
-
-   The list of protocols this node will support.
+   :values: @boollist
 
-   It takes a space-separated list of protocols (which are configured
-   separately) as its value.
+   The list of protocols to enable.
```

Comment:
so how do you enable **just** one protocol?

Reply:
Same as before, `protocols = imap` - list can be just one. So not sure what could be clarified here?

Reply:
it could show an example here for both. because now all around documentation we only have the incremental one.

Reply:
How about now?